avocado/core/varianter.py

```diff
@@ -346,3 +346,6 @@ def itertests(self):
             yield {"variant": self._default_params.get_leaves(),
                    "variant_id": None,
                    "paths": ["/run/*"]}
+
+    def __len__(self):
+        return self._no_variants
```

Comment:
Hmm, shouldn't this also raise exception in case `_no_variants` is `None`? Well for now I'm fine with this as it should be obvious that `None` is not really good value.